### PR TITLE
Apply some basic CI testing

### PR DESCRIPTION
**Pull request by glitchcrab**

### CODE REVIEW

File: kube-app-testing.sh [hunk 1]
```diff
@@ -596,18 +587,16 @@ create_gs_cluster () {
   info "Testing tenant cluster by listing pods in 'kube-system' namespace."
   # test connectivity
   kubectl get pods -n kube-system
```
Comment (piontec) [inline kube-app-testing.sh:589]: It seems you don't need this one after the change in the next line

**Comment by piontec** [comment]
ping @glitchcrab , what do we do about this one?

**Comment by glitchcrab** [comment]
It's not quite done yet - I think I forgot about this as we weren't going to invest any more time into this script. My plan was to use shellcheck to test the script to ensure it at least had some basic CI applied. Do you you think it is worth doing this @piontec?

**Comment by piontec** [comment]
I'm actually not sure... if that's a lot of time, it's not worth it, let's better focus on bringing the same functionality to kube-app-testing

**Comment by glitchcrab** [comment]
closing - i don't see the point in spending more time on this than is necessary. this pr is just a nice-to-have.

**Pull request closed without merging**